example/src/main/java/org/apache/flink/FlinkConnectorSourceExample.java

```diff
@@ -129,7 +133,8 @@ public static void nebulaVertexStreamSource() throws Exception {
         StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
         env.setParallelism(1);
 
-        NebulaSourceFunction sourceFunction = new NebulaSourceFunction(storageConnectionProvider)
+        NebulaSourceFunction sourceFunction = new NebulaSourceFunction(storageConnectionProvider,
+                metaConnectionProvider)
```

Comment:
This will change the interface used by users, can we init the metaConnectionProvider inside?